build.sbt

```diff
@@ -133,7 +134,7 @@ lazy val api = project
     Libraries.tapirOpenApi,
     Libraries.http4sServer
   ))
-  .dependsOn(core)
```

Comment:
I think one service should not depend on another. Move common data in core module or create new one if necessary